io.catenax.shared.message_header/2.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,140 @@
+   samm:properties (:messageId :context :sentDateTime :senderBpn :receiverBpn [ samm:property :expectedResponseBy; samm:optional true ] [ samm:property :relatedMessageId; samm:optional true ] :version).
+
+:messageId a samm:Property;
+   samm:preferredName "Message ID"@en;
+   samm:description "Unique ID identifying the message. The purpose of the ID is to uniquely identify a single message, therefore it MUST not be reused."@en;
+   samm:characteristic :IdTrait;
+   samm:exampleValue "3b4edc05-e214-47a1-b0c2-1d831cdd9ba9".
+
+:context a samm:Property;
+   samm:preferredName "Context"@en;
+   samm:description "Information about the context the message should be considered in.\nThe value MUST consist of two parts: an identifier of the context (e.g. business domain, etc.) followed by a version number.\nBoth the identifier and the version number MUST correspond to the content of the message.\nIf the content of a message is described by an aspect model available in the Catena-X Semantic Hub, then the unique identifier of this semantic model (e.g. urn:samm:io.catenax.<ASPECT-MODEL-NAME>:1.x.x) MUST be used as a value of the context field. This is considered the default case.\nIn all other cases the value of the context field MUST follow the pattern <domain>-<subdomain>-<object>:<[major] version> (e.g. TRACE-QM-Alert:1.x.x).\nVersioning only refers to major versions in both default and fallback cases.\nNote: The version of the message's header is specified in the version field."@en;
+   samm:characteristic :ContextCharacteristic;
+   samm:exampleValue "urn:samm:io.catenax.<ASPECT-MODEL-NAME>:1.x.x".
+
+:sentDateTime a samm:Property;
+   samm:preferredName "Sent Date Time"@en;
+   samm:description "Time zone aware timestamp holding the date and the time the message was sent by the sending party. The value MUST be formatted according to the ISO 8601 standard"@en;
+   samm:characteristic samm-c:Timestamp;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTime.
+
+:senderBpn a samm:Property;
+   samm:preferredName "Sender BPN"@en;
+   samm:description "The Business Partner Number of the sending party. The value MUST be a valid BPN. BPNA and BPNS are not allowed. Applicable constraints are defined in the corresponding standard"@en;
+   samm:characteristic :BpnCharacteristic;
+   samm:exampleValue "BPNL7588787849VQ".
+
+:receiverBpn a samm:Property;
+   samm:preferredName "Receiver BPN"@en;
+   samm:description "The Business Partner Number of the receiving party. The value MUST be a valid BPN. BPNA and BPNS are not allowed. Applicable constraints are defined in the corresponding standard."@en;
+   samm:characteristic :BpnCharacteristic;
```

Comment:
Since we re updating the model, let's use shared model for BPN Characteristic.

Reply:
Done :)